test-utils/user-api-client.ts

```
@@ -0,0 +1,187 @@
+    const services = (await response.json()).services;
+    const serviceOfferings: ServiceOffering[] = [];
+
+    // Include only Playwright Services
+    services.forEach((service) => {
+      service.offerings.forEach((offering) => {
+        const offeringData = {
+          ...service,
+          ...offering,
+        };
+
+        delete offeringData.offerings;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)